app/src/main/java/net/squanchy/notification/NotificationCreator.java

```diff
@@ -173,6 +174,7 @@ private String getDisplayedSpeakers(Event event) {
     }
 
     private String createSummaryTitle(int talksCount) {
-        return context.getString(R.string.event_notification_count_starting, talksCount);
+        String template = context.getString(R.string.event_notification_count_starting);
```

Comment:
I just think it does make sense to format it with the context method, because if the string is translated, it makes sense to use the locale that belongs to that translation. While if the string is hardcoded (e.g. a template) it doesn't make sense to change that with the locale (I'd argue we probably shouldn't use `String.format` for those cases but oh well, waiting for kotlin there :P)